### PR TITLE
new feature: display overall concern for each customer

### PR DESCRIPTION
**Pull request by petmetic**
To make this feature,  we had to implement a new column in the Customer model, because the `main concern` is linked to a specific Customer not the specific massage.
For easier access th the feature, you can edit it via the `Edit button` on the `customer/edit` page or in the `massage/edit` page via the `Edit button`.

### CODE REVIEW

File: src/web/templates/web/index.html [hunk 1]
```diff
@@ -20,9 +20,13 @@ <h4 class="fw-lighter">{% translate "Appointments for" %}: <span
                         {% for massage in massages %}
 
                             <li class="list-group-item d-flex justify-content-between align-items-center">
-                                <span class="fw-lighter"><span class="fw-normal uppercase">{{ massage.customer.name }} {{ massage.customer.surname }}</span> {% translate "at" %}
+                                <div><span class="fw-lighter"><span
+                                        class="fw-normal uppercase">{{ massage.customer.name }} {{ massage.customer.surname }}</span> {% translate "at" %}
                                     {{ massage.start|time:"H:i" }}
                                 </span>
+                                    <br>
+                                    <span class="fw-lighter">{{ massage.customer.main_concern }}</span>
```
Comment (jurecuhalev) [inline src/web/templates/web/index.html]: Do you want to show empty `<span>` and `<br />` if you don't have this info?

File: src/web/tests.py [hunk 2]
```diff
@@ -325,6 +325,160 @@ def test_check_listing_massage_views(self):
...
+                                ),
+                            },
+                        ],
+                    },
+                },
+            },
+        }
+
+        customer_import(data)
+        massage_import(data)
```
Comment (jurecuhalev) [inline src/web/tests.py:468]: Why does this test need importer at all, why not just use Customer Factory?

File: src/web/forms.py [hunk 3]
```diff
@@ -496,6 +509,7 @@ class Meta:
             "salon_choice",
             "frequency",
             "referral",
+            "main_concern",
```
Comment (jurecuhalev) [inline src/web/forms.py:528]: Where are tests and changes to customer edit html?

File: src/web/tests.py (hunk 2 above)
Comment (petmetic) [reply src/web/tests.py:468]: needs discussion...

File: src/web/views.py [hunk 4]
```diff
@@ -22,18 +22,26 @@
...
-    ).order_by("start")
+    massages = (
+        Massage.objects.filter(
+            therapist=therapist,
+            start__year=today.year,
+            start__month=today.month,
+            start__day=today.day,
+            status="approved",
+        )
+        .prefetch_related("customer")
```
[the remark this answers is not in the document]
Comment (petmetic) [reply src/web/views.py:33]: yes, `select_related` would be better for this search beacuse it selects additional obect related data, while `prefetch_related` does a seperate query for each lookup and a joining in python (is faster this way)...
and yes, maybe I don't need `select_related` at all? It works without it... but is it faster with the `select_related`?

**Review by jurecuhalev — APPROVED** [review]

**Pull request merged**